Tests/Rewriters/ObjectRewriterTests.cs

```diff
@@ -263,10 +290,58 @@ public void Rewrite_Initializer (string source, string expected)
         @"ITestInterface noMock = _noMock;",
         //language=C#
         @"ITestInterface noMock = _noMock;")]
+    public void Rewrite_LocalDeclarationStatement (string source, string expected)
+    {
+      var (model, node) = CompiledSourceFileProvider.CompileLocalDeclarationStatementWithContext (source, _context, true);
+      var (_, expectedNode) = CompiledSourceFileProvider.CompileLocalDeclarationStatementWithContext (expected, _context);
+      _rewriter.Model = model;
+      _rewriter.RhinoMocksSymbols = new RhinoMocksSymbols (model.Compilation);
+      _rewriter.MoqSymbols = new MoqSymbols (model.Compilation);
+      var actualNode = _rewriter.Visit (node);
+
+      Assert.NotNull (actualNode);
+      Assert.That (expectedNode.IsEquivalentTo (actualNode, false));
+    }
+
+    [Test]
+    [TestCase (
+        //language=C#
+        @"public Mock<ITestInterface> parentMock = new Mock<ITestInterface>();",
+        //language=C#
+        @"public Mock<ITestInterface> parentMock = new Mock<ITestInterface>();")]
+    [TestCase (
+        //language=C#
+        @"public ITestInterface parentMock = new Mock<ITestInterface>();",
+        //language=C#
+        @"public ITestInterface parentMock = new Mock<ITestInterface>().Object;")]
+    public void Rewrite_FieldDeclaration (string source, string expected)
+    {
+      var (model, node) = CompiledSourceFileProvider.CompileFieldDeclarationWithContext (source, _context, true);
+      var (_, expectedNode) = CompiledSourceFileProvider.CompileFieldDeclarationWithContext (expected, _context);
+      _rewriter.Model = model;
+      _rewriter.RhinoMocksSymbols = new RhinoMocksSymbols (model.Compilation);
+      _rewriter.MoqSymbols = new MoqSymbols (model.Compilation);
+      var actualNode = _rewriter.Visit (node);
+
+      Assert.NotNull (actualNode);
+      Assert.That (expectedNode.IsEquivalentTo (actualNode, false));
+    }
+
+    [Test]
+    [TestCase (
+        //language=C#
+        @"_noMock = new Mock<ITestInterface>();",
+        //language=C#
+        @"_noMock = new Mock<ITestInterface>().Object;")]
+    [TestCase (
+        //language=C#
+        @"_mock = new Mock<ITestInterface>();",
+        //language=C#
+        @"_mock = new Mock<ITestInterface>();")]
     public void Rewrite_AssignmentExpression (string source, string expected)
     {
-      var (model, node) = CompiledSourceFileProvider.CompileExpressionStatementWithContext (source, _context, true);
-      var (_, expectedNode) = CompiledSourceFileProvider.CompileExpressionStatementWithContext (expected, _context);
+      var (model, node) = CompiledSourceFileProvider.CompileAssignmentExpressionWithContext (source, _context, true);
```

Comment:
Those are statements. Wouldn't it be possible to reduce the used method to `CompileStatementWithContext`?